include/tvm/relay/expr.h

```diff
@@ -373,6 +373,9 @@ class IfNode : public ExprNode {
 
 RELAY_DEFINE_NODE_REF(If, IfNode, Expr);
 
+struct Environment;
+std::ostream & DebugPrint(const Environment &, const Expr &, std::ostream & os);
```

Comment:
Could you move the forward declaration to a common location say `base.h`?

Also can you add documentation for this for example.

`/*! \brief Print a debug representation of the expression to the stream.
    *   \param env The environment.
    *  \params ...
    *   \returns A reference to the stream.
    */
```